README.md

```diff
@@ -116,7 +121,8 @@ We recommend running the pipeline on a dedicated [Cromwell](https://github.com/b
 > cp $GATK_SV_ROOT/wdl/*.wdl .
 > zip dep.zip *.wdl
 > cd ..
-> bash scripts/inputs/build_default_inputs.sh -d $GATK_SV_ROOT
+> echo '{ "google_project_id": "my-google-project-id" }' > inputs/values/google_cloud.my_project.json
```

Comment:
Would it be a little easier to just provide the project directly to the build script? ie. `-c "broad-dsde-methods"`. I tested this process and it seems to work fine, but I wonder if it would not be easier to just directly supply the project rather than creating a file

Reply:
It would be a little simpler to do that. I think we should keep it as a file definition to remain consistent with how all inputs are defined. It also makes it clear/intuitive that the google project id is a workflow input and as well as that it is not somehow being used for the build process.